elevate-cpanel

```diff
@@ -832,7 +832,8 @@ EOS
         /scripts/elevate-cpanel --check
 
         If your GRUB2 configuration has not been customized, consider reporting
-        this limitation to https://github.com/cpanel/elevate/issues
+        this limitation to cPanel Support:
```

Comment:
maybe not reporting? I can't find a good phrase :/

Reply:
"you may want to consider reaching out to cPanel Support for assistance" ?